### DISCUSSION TITLE
Monorepo without workspaces

### DISCUSSION THREAD

**Discussion by jeffreyyoung** [vercel/turborepo#4733] [Help] [answered]
Hi!

I have multiple javascript projects in a single folder.  Each product is independent of the other and there are no inter-project dependencies.  I am wondering if it is possible to use turbo repo in this "monorepo", without enabling yarn workspaces or anything similar?  I am mainly interested in cached builds and being able to build all projects with a single command

Thank you!

**Comment by Amoodaa** [comment, marked as answer]
@jeffreyyoung the benefit of workspaces is to automatically infer the scope of files under a package,
If you dont want that, you can alter the behaviour manually using the inputs,
you need to define a build/dev script for each "project" 
as well as its inputs and outputs,
this should get you going: https://turbo.build/repo/docs/core-concepts/caching#alter-caching-based-on-file-changes
